lib/mappers/__tests__/mapper.figma.test.js

```diff
@@ -0,0 +1,44 @@
+const getFixtures = require('./__fixtures__/mapper.figma');
+const getMapper = require('..');
+
+let FIXTURES = {};
+let mappers;
+
+describe('figma mapper test suite', () => {
+  beforeAll(() => {
+    mappers = getMapper(false); // false means no sketchfile
```

Comment:
What if we later decide to support Adobe XD? You can only provide true/false now.

Deciding how to map should be the responsibility of the getMapper function. You should just pass `file` to the mapper where it can decide what type of file it is.

Reply:
Why are these not just globally scoped? Do they need to be reassigned to the mutable values before every test?

Just create 2 consts at the top and reference those? I don't see any reason why reassigning these values is required.